server/middleware/setUpWebSession.ts

```diff
@@ -1,19 +1,26 @@
 import { v4 as uuidv4 } from 'uuid'
-import session from 'express-session'
+import session, { MemoryStore, Store } from 'express-session'
 import RedisStore from 'connect-redis'
 import express, { Router } from 'express'
 import { createRedisClient } from '../data/redisClient'
 import config from '../config'
 import logger from '../../logger'
 
 export default function setUpWebSession(): Router {
-  const client = createRedisClient()
-  client.connect().catch((err: Error) => logger.error(`Error connecting to Redis`, err))
+  let store: Store
+  if (config.redis.enabled) {
+    const client = createRedisClient()
+    client.connect().catch((err: Error) => logger.error(`Error connecting to Redis`, err))
+    store = new RedisStore({ client })
+  } else {
+    store = new MemoryStore()
+  }
 
   const router = express.Router()
   router.use(
     session({
-      store: new RedisStore({ client }),
+      store,
+      name: 'hmpps-digital-prison-services.session',
```

Comment:
We didn't have a session name set before so this will invalidate people's current session - I don't think that will be a problem.  Useful to have though, in case we have that issue where someone else's service extends their cookie reach across `justice.gov.uk`